continuous_integration/docker/base/Dockerfile

```diff
@@ -1,33 +1,79 @@
-FROM centos:7.6.1810
+# See continuous_integration/docker/README.md for details about this and other
+# Dockerfiles under the continuous_integration/docker folder on their purpose
+# and how to work with them.
+#
+# centos:8 reached end-of-life 31 Dec 2021
+# centos:7 reach end-of-life 30 Jun 2024
```

Comment:
I figure it is like ubuntu 20.04 vs 21.10, one is LTS, one isnt. Do you choose the latest and get weaker software support or the older with better?

I wanted to be compatible with software without needing to re-compile it, centos 7 vs centos 8 comes with ups and downs. Various versions with PBS supported various versions of centos etc. Since centos 8 has reached end of life already, i heaitated to bump to that as it probably would fail to be compatible with software going onwards even if it didnt yet.

There are even more modern versions i think, but overall this seems reasonable to stick with to reduce complexity and avoid work.